src/controllers/itemdetailpage.js

```diff
@@ -1379,10 +1387,8 @@ define(["loading", "appRouter", "layoutManager", "connectionManager", "cardBuild
                 userId: userId,
                 Fields: fields
             });
-        } else if (!("MusicAlbum" == item.Type)) {
-            if ("MusicArtist" == item.Type) {
-                query.SortBy = "ProductionYear,SortName";
-            }
+        } else if ("MusicAlbum" != item.Type && "MusicArtist" == item.Type) {
```

Comment:
```js
} else if ("MusicArtist" == item.Type) {
```
no need to check if `item.Type` isn't Album when we compare it to Artist next

Reply:
https://github.com/jellyfin/jellyfin-web/pull/509/commits/47cc029c7056b2ad334a93ba80f1568c6020a0ad